layouts/partials/header.html

```diff
@@ -1,11 +1,16 @@
 <header class="header">
 	<div class="container">
-		<div class="logo" role="banner">
-			<a class="logo__link" href="{{ "" | relLangURL }}" title="{{ .Site.Title }}" rel="home">
+		<a class="logo" href="{{ relLangURL "" }}" title="{{ .Site.Title }}" rel="home" role="banner">
+			{{ with .Site.Params.logo }}
+			<div>
```

Comment:
Why div? Why not just img without extra wrapper?

Reply:
Because it caused weird behaviour without the div. Somehow the flexbox failed.

Reply:
OK, I'll check it tomorrow.

Reply:
> The image becomes deformed

Yep. I'll see what can be done with this.

Reply:
May I ping you on this one? It seems that this was the only point blocking another review.

Reply:
Another ping: the div around img looks weird but seems to be necessary to preserve svg aspect ratio. Can we leave it as is?